### ISSUE TITLE
Change the name of the repo

### ISSUE THREAD

**Issue by adamralph** [poteto/hiring-without-whiteboards#410]
Whiteboards are not the problem. It's what you are asked to in an interview (with or without) a whiteboard that is the problem. You even say in the readme:

> - Discussing a real world problem with the help of a whiteboard is 👍
> - Solving basic CS questions, technical puzzles, riddles, brainteasers (with or without whiteboard) is 👎

Poor whiteboards are getting drawn into this for no good reason.

(FWIW - as an interviewee, I love having a whiteboard handy to illustrate my points.)

**Comment by samelamin** [comment]
Agreed, it would be better to say hiring without CS Algo questions 

Illustrating on a whiteboard is the best way to visualise and help explain a problem

**Comment by poteto** [comment]
I'll admit that the title was meant to be clickbaity. Whether or not the interview uses a whiteboard, I think we can generally agree that whiteboards in interview _tend_ to have negative connotations that correlate to answering CS trivia. So I thought "Hiring Without Whiteboards" would be apt.

Might there be a better name that is shorter?

**Comment by samelamin** [comment]
How about "hiring without CS algorithms?"?

**Comment by jaredreich** [comment]
"hiring-reasonably"

**Comment by braham-snyder** [comment]
> ... negative connotations that correlate to answering CS trivia.

> Solving CS trivia ...  (with or without whiteboard) is 👎

"trivia" is the most succinct descriptor I've encountered of (my conception of) the hiring practices in question, so my vote's for something along the lines of "hiring without trivia"

**Comment by SuperSpyTX** [comment]
"modern-interviews"?

**Comment by milieuChris** [comment]
Very much agree. Depending on the experience and expectations of a potential client i often give them a choice to either describe verbally, or visually on a whiteboard, or some combination. What I'm looking for is that they are first able to communicate, and subsequently to learn a bit about their preferred styles of communication.
